### PR TITLE
Polish BERT example

### PR DESCRIPTION
**Pull request by gpengzhi**
Update some outdated code and description.

### CODE REVIEW

File: examples/bert/README.md [hunk 1]
```diff
@@ -27,63 +31,72 @@ pip install -r requirements.txt
...
 ### Prepare data
 
-We first preprocess the downloaded raw data into [TFRecord](https://www.tensorflow.org/tutorials/load_data/tf_records) files. The preprocessing tokenizes raw text with BPE encoding, truncates sequences, adds special tokens, etc.
-Run the following cmd to this end: 
-```
-    python prepare_data.py --task=MRPC
-    [--max_seq_length=128]
-    [--vocab_file=bert_pretrained_models/uncased_L-12_H-768_A-12/vocab.txt]
-    [--tfrecord_output_dir=data/MRPC] 
+We first preprocess the downloaded raw data into [pickle](https://docs.python.org/3/library/pickle.html) files. The preprocessing tokenizes raw text with BPE encoding, truncates sequences, adds special tokens, etc. Run the following cmd to this end:
```
Comment (huzecong) [inline examples/bert/README.md]: They're actually called "pickled files".

File: examples/bert/README.md [hunk 2]
```diff
@@ -27,63 +31,72 @@ pip install -r requirements.txt
...
 
 **Outcome of the Preprocessing**:
-- The preprocessing will output 3 TFRecord data files `{train.tf_record, eval.tf_record, test.tf_record}` in the specified output directory.
+
+- The preprocessing will output 3 pickle data files `{train.pickle, eval.pickle, test.pickle}` in the specified output directory.
+
 - The cmd also prints logs as follows:
-  ```
+
+  ```bash
```
Comment (huzecong) [inline examples/bert/README.md]: Let's not treat logs as bash code -- they shouldn't be highlighted.

File: examples/bert/config_data.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "dataset": {
         "data_name": "data",
         "feature_original_types": feature_original_types,
-        "files": "{}/train.tf_record".format(tfrecord_data_dir)
+        "files": "{}/train.pickle".format(pickle_data_dir)
```
Comment (huzecong) [inline examples/bert/config_data.py]: Pickled files usually have extension names of `pkl`.

File: examples/bert/README.md [hunk 4]
```diff
@@ -27,63 +31,75 @@ pip install -r requirements.txt
...
+--output_dir=data/MRPC
 ```
+
 - `task`: Specifies the dataset name to preprocess. BERT provides default support for `{'CoLA', 'MNLI', 'MRPC', 'XNLI', 'SST'}` data.
 - `max_seq_length`: The maxium length of sequence. This includes BERT special tokens that will be automatically added. Longer sequence will be trimmed. 
-- `vocab_file`: Path to a vocabary file used for tokenization.
-- `tfrecord_output_dir`: The output path where the resulting TFRecord files will be put in. Be default, it is set to `data/{task}` where `{task}` is the (upper-cased) dataset name specified in `--task` above. So in the above cmd, the TFRecord files are output to `data/MRPC`.
+- `pretrained_model_name`: The name of a pre-trained model to load selected in the list of: `bert-base-uncased`, `bert-large-uncased`, `bert-base-cased`, `bert-large-cased`, `bert-base-multilingual-uncased`, `bert-base-multilingual-cased`, and `bert-base-chinese`.
+- `output_dir`: The output path where the resulting pickled files will be put in. Be default, it is set to `data/{task}` where `{task}` is the (upper-cased) dataset name specified in `--task` above. So in the above commend, the pickled files are output to `data/MRPC`.
```
Comment (ZhitingHu) [inline examples/bert/README.md]: "commend" --> "command"

Comment (ZhitingHu) [reply examples/bert/README.md]: It looks "commend" occurs in multiple places in the README

Comment (gpengzhi) [reply examples/bert/README.md]: My bad.

**Comment by gpengzhi** [comment]
@ZhitingHu Yes, I reproduced the previous results.

**Pull request merged**